tests/features/multisite_wysiwyg/rich_text_editing.feature

```diff
@@ -0,0 +1,26 @@
+@api
+Feature: Testing the rich text options available with the toolbar present on
+  WYSIWYG fields like "Body".
```

Comment:
When possible I try to respect the rule on the max. of characters per line (80 per line).